README.md

```diff
@@ -10,6 +10,33 @@ Fake for Seam Devicedb. Use with Fake Seam Connect.
 This module allows you to spin up an in-memory server
 for the Seam Devicedb.
 
+## 📓 Notes
+
+### Text Search
+
+Full-text search on real devicedb API (using the `?text_search` parameter) is far more flexible than the simple matching provided by this fake.
+
+### Seeding with Data
+
+This fake can be seeded with data from the live API. To do so, either call `POST /_fake/populate-from-live-api` or use the exported helper:
+
+```ts
+import { createDatabase, populateFromLiveApi } from "@seamapi/fake-devicedb"
+
+const db = createDatabase()
+
+await populateFromLiveApi({
+  db,
+  vercel_protection_bypass_secret: "<secret>",
+  endpoint: "https://devicedb.seam.tube",
+  device_category: "smartlock",
+})
+```
```

Comment:
@razor-x look good?

Reply:
If we could populate via seam connect we would not need to document this `vercel_protection_bypass_secret`. The interface here could be

```
await populateFromLiveApi({
  db,
  client: axios.create({ headers: {...}),
  device_category: "smartlock",
})
```

Reply:
The main reason for this though is if I can pass a client, I can use one configured with a publishable key and then this step can be added in npm script in the react repo with needing any special permissions.

Reply:
I think that could work, but it would still require a client session / API key so I'm not sure what's more ergonomic to use if this is called manually in a script. When do you plan on calling it?

Reply:
whoops didn't see your second comment

Reply:
yeah I think that works

Reply:
Yea, in practice this will be used like this with the new client (also realized this method should be added to the fake, not exported, since we have `fake.seed()` it makes send to have `fake.seedFromLikeApi`

```ts
const seam = SeamHttp.fromPublishableKey(...)

await fake.seedFromLiveApi(seam.client)
```

Reply:
👍 going to keep it as exported rather than a method on the fake since imo it's more of a helper and I don't want to pollute the standard fake interface

Reply:
e35b652